frontends/concrete-python/Makefile

```diff
@@ -39,7 +39,17 @@ licenses:
 # Testing
 # =======
 
-pytest: pytest-single pytest-multi
+pytest: pytest-default
```

Comment:
why an alias ?

Reply:
just because we maybe want to test other options later...